scripts/test_antlr_grammar.sh

```diff
@@ -67,11 +63,11 @@ test_file()
   local output
   if [[ "${solOrYul}" == "sol" ]]; then
     output=$(
-      java \
+      cat "${SOL_FILE}" | grep -v "^==== ExternalSource:" | java \
```

Comment:
Maybe this will fix coding style problem?
```suggestion
      < "${SOL_FILE}" | grep -v "^==== ExternalSource:" | java \
```

Reply:
I'd go for just passing the file directly to grep:
```suggestion
      grep -v "^==== ExternalSource:" "${SOL_FILE}" | java \
```

Reply:
Passing file to grep sounds good to me as well

Reply:
Yep, definitely.